HyperBrain/serverless-vscode
# Use global serverless executable if config option is set

## What did you implement:

Closes #1



## How did you implement it:



If the `serverless.useGlobal` option is set to true, the extension will try to spawn a globally available `serverless` command.

## How can we verify it:



Install serverless globally with `npm install -g serverless` and set the workspace option `serverless.useGlobal: true`.
Then execute any serverless command via the context menu. The command should run and
use the global serverless installation.

## Todos:


***Is this ready for review?:*** NO
***Is it a breaking change?:*** NO
